doc/source/whatsnew/v0.18.0.txt

```diff
@@ -348,4 +349,6 @@ Bug Fixes
 
 - Bug in ``read_sql`` with pymysql connections failing to return chunked data (:issue:`11522`)
 
+- Bug in ``.to_csv`` ignoring formatting parameters `decimal`, `na_rep`, `float_format` for float indexes (:issue:`11553`)
```

Comment:
use double back-ticks (instead of the single back ticks) around decimal, na_rep, float_format